src/main/resources/Azure_Instance_Details.csv

```diff
@@ -1,89 +1,88 @@
 instance,API_name,vCPUs,Memory_GB,Compute_Contract_Price,On_Demand_Cost_Hourly,Linux_Reserved_Cost_Hourly,Hourly_DBUs

```

Comment:
Here, it looks like this is a full copy / paste. I cannot assure that only one line has changed, please only remove the one offending line, not copy and paste an entirely new document.

I also see another issue, you have removed a fix I implemented earlier. Notice how on line 2, Memory has moved back from 28.0 to 28. 28 gets interpreted as an Integer and causes issues, we need it to be a double. 

Please just drop the one offending line.